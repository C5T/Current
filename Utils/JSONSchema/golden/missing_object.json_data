[
  {"x":{"a":"b"}},
  {"x":null}
]
