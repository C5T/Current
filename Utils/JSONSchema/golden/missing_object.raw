{"ObjectOrArray":{"fields":{"":{"ObjectOrArray":{"fields":{"x":{"ObjectOrArray":{"fields":{"a":{"String":{"values":{"b":1},"instances":1,"nulls":0}}},"instances":1,"nulls":1}}},"instances":2,"nulls":0}}},"instances":1,"nulls":0}}