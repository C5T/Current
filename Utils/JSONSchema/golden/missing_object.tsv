Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	2	0	1 field	x
Schema[].x	Object	1	1	1 field	a
Schema[].x.a	String	1	0	1 distinct value	b : 1
